designs/dedicatedhosts.md

```diff
@@ -0,0 +1,86 @@
+# Dedicated host support
+## Background
+
+As described in [3182](https://github.com/aws/karpenter/issues/3182) AWS provides the 
+ability to launch EC2 instances on [Dedicated Host hardware](https://docs.aws.amazon.com/license-manager/latest/userguide/host-resource-groups.html)
+Dedicated hosts are often used to pay for ec2 usage as capital expenditure (CapEx)
```

Comment:
do savings plans and RIs also allow for this? I thought dedicated hosts were mainly an avenue to deal with software licenses?

Reply:
yeah SP and RIs also allow for this, but DH are a flexible mechanism to both place controls around usage, can be shared between accounts with RAM and can be prepaid (like SPs and RIs).

There was another specific reason why we picked DH over RIs - I just can't think of it right now :P I'll have to get back to you on this.